hack/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2023.
```

Comment:
please also add the boilerplate to the go files

Reply:
done :)